src/guides/v2.3/extension-dev-guide/staging.md

```diff
@@ -6,6 +6,9 @@ ee_only: true
 
 Staging functionality enables you to create scheduled campaigns, manage campaigns on dashboard and manipulate each future store object update. Moreover, you can preview a [storefront](https://glossary.magento.com/storefront) during the future campaign.
 
+{:.bs-callout-info}
```

Comment:
```suggestion
{:.bs-callout-tip}
```

Reply:
I'm good with that! Thanks for the tip [sic]